gitpod-io/gitpod
# [JetBrains] Remove JDK auto-setter

Because JDK should be correctly detected now that https://github.com/gitpod-io/gitpod/pull/14916 has been merged.

## Description

_To be written..._

## Related Issue(s)



## How to test

Test InteliJ IDEA (Stable and EAP) with these 3 branches:
- https://github.com/Gitpod-Samples/spring-petclinic/tree/master
- https://github.com/Gitpod-Samples/spring-petclinic/tree/update-idea-folder
- https://github.com/Gitpod-Samples/spring-petclinic/tree/delete-idea-folder

And confirm in all of them, the JDK is set correctly. (From Top Menu: File >> Project Structure)

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
